### PR TITLE
adds Arch Linux package to doc/install

### PR DESCRIPTION
**Pull request by wookietreiber**
I have just added an Arch Linux package for mill [here](https://aur.archlinux.org/packages/mill/), which I included in the docs. I have also added some headers to distinguish the different installation methods. Feel free to change the headers if you don't like them as is.

### CODE REVIEW

File: docs/pages/1 - Intro to Mill.md [hunk 1]
```diff
@@ -15,19 +15,35 @@ programming language, and can serve as a replacement for
...
 external subprocesses.
 
+## Installation
+
+### Arch Linux
+
+Arch Linux has an [AUR package for mill](https://aur.archlinux.org/packages/mill/):
+
+```bash
+pacaur -S mill
```
Comment (rockjam) [inline docs/pages/1 - Intro to Mill.md]: or `yaourt -S  mill` I guess ?

Comment (rockjam) [reply docs/pages/1 - Intro to Mill.md]: is `pacaur ` is some newer alternative to yaourt?

**Comment by rockjam** [comment]
@wookietreiber currently mill doesn't work on java 9. Is there a way to restrict java version in `PKGBUILD` for now?

File: docs/pages/1 - Intro to Mill.md (hunk 1 above)
Comment (wookietreiber) [reply docs/pages/1 - Intro to Mill.md]: Yes, `pacaur` is newer and IMO the better alternative.

**Comment by wookietreiber** [comment]
@rockjam Yes, we can restrict, doc is [here](https://wiki.archlinux.org/index.php/PKGBUILD#depends). I have added this to the package.

**Comment by rockjam** [comment]
Thank you

**Pull request merged**